bazel/example/main.cc

```diff
@@ -9,7 +9,8 @@
 using ::google::protobuf::TextFormat;
 using ::p4::config::v1::P4Info;
 
-using namespace P4::literals;
+using namespace ::P4;
+using namespace ::P4::literals;
```

Comment:
We could turn the `literals` nested namespace into inline namespace now. That way, anyone who does `using namespace P4` would get the literals. But that should be a separate task.

Reply:
Actually, yes! Completely forgot about this.